src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.58.1'
+__version__ = '1.59.0'
```

Comment:
This is a patch version bump because there is no new feature

Reply:
Gotcha. I wasn't sure if adding documentation counted as a new feature. Makes sense though.

Reply:
Especially this documentation which was supposed to be there but was missing.